<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>MainWindow</class>
 <widget class="QMainWindow" name="MainWindow">
  <property name="windowTitle">
   <string>MainWindow</string>
  </property>
  <widget class="QWidget" name="centralwidget">
   <widget class="QLabel" name="label_vehicle_label">
    <property name="geometry">
     <rect>
      <x>30</x>
      <y>20</y>
      <width>201</width>
      <height>91</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>16</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Vehicle Department:</string>
    </property>
   </widget>
   <widget class="QTableWidget" name="table_bus">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>150</y>
      <width>691</width>
      <height>101</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Brand</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Service begin (date)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Fuel consumption per 1km</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mileage</string>
     </property>
    </column>
   </widget>
   <widget class="QTableWidget" name="table_tram">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>290</y>
      <width>691</width>
      <height>101</height>
     </rect>
    </property>
    <column>
     <property name="text">
      <string>Brand</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Service begin (date)</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Electricity consumption per hour</string>
     </property>
    </column>
    <column>
     <property name="text">
      <string>Mileage</string>
     </property>
    </column>
   </widget>
   <widget class="QLabel" name="label_bus">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>120</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Buses info:</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_tram">
    <property name="geometry">
     <rect>
      <x>50</x>
      <y>260</y>
      <width>91</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>Trams info:</string>
    </property>
   </widget>
   <widget class="Line" name="line">
    <property name="geometry">
     <rect>
      <x>-10</x>
      <y>100</y>
      <width>811</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="Line" name="line_2">
    <property name="geometry">
     <rect>
      <x>0</x>
      <y>400</y>
      <width>821</width>
      <height>16</height>
     </rect>
    </property>
    <property name="orientation">
     <enum>Qt::Horizontal</enum>
    </property>
   </widget>
   <widget class="QLabel" name="label_new_title">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>420</y>
      <width>141</width>
      <height>16</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>New vehicle</string>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_brand">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>450</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_milage">
    <property name="geometry">
     <rect>
      <x>100</x>
      <y>490</y>
      <width>171</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QLineEdit" name="line_consumption">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>450</y>
      <width>111</width>
      <height>21</height>
     </rect>
    </property>
   </widget>
   <widget class="QDateEdit" name="date_service">
    <property name="geometry">
     <rect>
      <x>410</x>
      <y>490</y>
      <width>110</width>
      <height>22</height>
     </rect>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioBus">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>450</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Bus</string>
    </property>
   </widget>
   <widget class="QRadioButton" name="radioTram">
    <property name="geometry">
     <rect>
      <x>540</x>
      <y>490</y>
      <width>82</width>
      <height>17</height>
     </rect>
    </property>
    <property name="text">
     <string>Tram</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_brand_new">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>450</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Brand</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_milage_new">
    <property name="geometry">
     <rect>
      <x>20</x>
      <y>490</y>
      <width>47</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Mileage</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_consumption_new">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>450</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Consumption(100km)</string>
    </property>
   </widget>
   <widget class="QLabel" name="label_service_new">
    <property name="geometry">
     <rect>
      <x>300</x>
      <y>490</y>
      <width>101</width>
      <height>21</height>
     </rect>
    </property>
    <property name="text">
     <string>Service begin</string>
    </property>
   </widget>
   <widget class="QPushButton" name="button_create">
    <property name="geometry">
     <rect>
      <x>630</x>
      <y>460</y>
      <width>101</width>
      <height>41</height>
     </rect>
    </property>
    <property name="text">
     <string>Create</string>
    </property>
   </widget>
   <widget class="QLabel" name="label">
    <property name="geometry">
     <rect>
      <x>610</x>
      <y>0</y>
      <width>141</width>
      <height>101</height>
     </rect>
    </property>
    <property name="text">
     <string/>
    </property>
    <property name="pixmap">
     <pixmap>logos/bus1.png</pixmap>
    </property>
    <property name="scaledContents">
     <bool>true</bool>
    </property>
   </widget>
   <widget class="QLabel" name="label_location">
    <property name="geometry">
     <rect>
      <x>340</x>
      <y>40</y>
      <width>251</width>
      <height>51</height>
     </rect>
    </property>
    <property name="font">
     <font>
      <pointsize>14</pointsize>
     </font>
    </property>
    <property name="text">
     <string>LOCATION</string>
    </property>
   </widget>
  </widget>
  <widget class="QMenuBar" name="menubar">
   <property name="geometry">
    <rect>
     <x>0</x>
     <y>0</y>
     <width>777</width>
     <height>21</height>
    </rect>
   </property>
   <widget class="QMenu" name="menuLogout">
    <property name="title">
     <string>Menu</string>
    </property>
    <addaction name="action_back_to_menu"/>
    <addaction name="actionlogout"/>
   </widget>
   <addaction name="menuLogout"/>
  </widget>
  <widget class="QStatusBar" name="statusbar"/>
  <action name="actionlogout">
   <property name="text">
    <string>logout</string>
   </property>
  </action>
  <action name="action_back_to_menu">
   <property name="text">
    <string>Back to menu</string>
   </property>
  </action>
 </widget>
 <resources/>
 <connections/>
</ui>
